### DISCUSSION TITLE
Google Maps Officially Supported Plugin

### DISCUSSION THREAD

**Discussion by selected-pixel-jameson** [capacitor-community/google-maps#169] [General]
What does everyone else think about Ionic just dropping the news that they now have an officially supported plugin? Does it make sense for this to continue to be maintained? 

I don't know if anyone has been in contact at all with the Ionic team, but as far as I'm aware there has been no communications. I wonder how much of our code they ripped off. Would have bee nice if they at least through it out there and then we could have all worked together. 

Thank you @tafelnl and @hemangsk and all the other contributors for all your hard work on this.

**Comment by tafelnl** [comment]
They actually contacted me a couple of weeks ago. I asked them which approach they were going to take with regards to rendering the map behind the webview vs in front of the webview.

They said they were going for the approach were the map is rendered **in front of** the webview. I asked them why they choose to go that route, but they never responded. So to be honest, I forgot about it after that.

I am curious how their plugin works in comparison to this one. Rendering the map behind the webview is a big feature of this plugin, which allows for several use-cases that would not be possible otherwise.

I have not yet played around with their plugin yet, and so I have no clue what the differences are and how it works exactly. Did you test it out yet by any chance?

**Comment by selected-pixel-jameson** [comment]
Oh that's interesting! I didn't realize they were not rendering the map behind the Webview. They are going to run into immediate issues with that just like the community did. Users will want to add the overlay buttons to the maps. 

I wonder why they took that approach? Interesting that they wouldn't respond. Seems like they don't have a good grasp on what the community is actually looking for, which isn't a big surprise since it took them this long to even implement a maps plugin. 

It's interesting reading the docs for the API. They are specifically addressing some of the issues that we've already had to deal with. 

```
On Android, the map is rendered beneath the entire webview, and uses this component to manage its positioning during scrolling events. This means that as the developer, you must ensure that the webview is transparent all the way through the layers to the very bottom. In a typically Ionic application, that means setting transparency on elements such as IonContent and the root HTML tag to ensure that it can be seen. If you can’t see your map on Android, this should be the first thing you check.

On iOS, we render the map directly into the webview and so the same transparency effects are not required. We are investigating alternate methods for Android still and hope to resolve this better in a future update.
```

Seems like the direction we are taking makes much more sense. Thanks again for all your effort.
